soc/arm/nxp_kinetis/k8x/Kconfig.soc

```diff
@@ -69,4 +69,6 @@ config K8X_FLASH_CLOCK_DIVIDER
 	  This option specifies the divide value for the K8x flash clock from the
 	  system clock.
 
+config COUNTER_MCUX_PIT
+	default y if HAS_MCUX_PIT
```

Comment:
Please move these changes to `soc/arm/nxp_kinetis/k8x/Kconfig.defconfig.series`

Reply:
ok